th2_data_services_lwdp/adapters/adapter_sse.py

```diff
@@ -32,28 +30,7 @@ def __init__(self, json_processor: BufferedJSONProcessor):
         self.json_processor = json_processor
         self.events_types_blacklist = {"close", "keep_alive", "message_ids"}
 
-    def handle(self, record: SSEEvent) -> dict:
-        """Adapter handler.
-
-        Args:
-            record: SSE Event.
-
-        Returns:
-            Dict object.
-        """
-        if record.event == "error":
-            raise HTTPError(record.data)
-        if record.event not in ["close", "keep_alive", "message_ids"]:
-            try:
-                return json.loads(record.data)
-            except json.JSONDecodeError as e:
-                raise ValueError(
-                    f"json.decoder.JSONDecodeError: Invalid json received.\n"
-                    f"{e}\n"
-                    f"{record.data}"
-                )
-
-    def handle_stream(self, stream: Iterable):
+    def handle(self, stream: Iterable):
         # We need this block because we put generator function in the commands.
         # TODO this hack will be removed when we add Data.map_stream
         if callable(stream):
```

Comment:
I had an idea to remove this piece of code and update command (check comment in http command)

What do you think?

Reply:
Todo Resolved